test/functionalTest/cases/117_convop_using_standard_ops/putSubscriptionConvOp.test

```diff
@@ -0,0 +1,763 @@
+echo "==========================================================="
+payload='{
+  "duration": "P1Y",
+  "subscriptionId": "1234567890",
+  "throttling": "PT1S",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+        "condValues": [
+          "A1"
+        ]
+    }
+  ]
+}'
+orionCurl --url /v1/contextSubscriptions/1234567890 --payload "$payload" --json -X PUT
+echo
+echo
+
+
+echo "02. PUT /v1/contextSubscriptions/123456789012345678901234 with 000000000000000000000000 in payload and see it fail"
+echo "=================================================================================================================="
+payload='{
+  "duration": "P1Y",
+  "subscriptionId": "000000000000000000000000",
+  "throttling": "PT1S",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+        "condValues": [
+          "A1"
+        ]
+    }
+  ]
+}'
+orionCurl --url /v1/contextSubscriptions/123456789012345678901234 --payload "$payload" --json -X PUT
+echo
+echo
+
+
+echo "03. PUT /v1/contextSubscriptions/123456789012345678901234 and see it fail"
+echo "========================================================================="
+payload='{
+  "duration": "P1Y",
+  "subscriptionId": "123456789012345678901234",
+  "throttling": "PT1S",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+        "condValues": [
+          "A1"
+        ]
+    }
+  ]
+}'
+orionCurl --url /v1/contextSubscriptions/123456789012345678901234 --payload "$payload" --json -X PUT
+echo
+echo
+
+
+echo "04. Subscribe to E1/T1/A1+A2, in XML"
+echo "===================================="
+payload='{
+  "entities": [
+    {
+        "id":   "E1",
+        "type": "T1"
+    }
+  ],
+  "attributes": [
+    "A1",
+    "A2"
+  ],
+  "reference": "http://localhost:'${LISTENER_PORT}'/notify",
+  "duration": "P1M",
+  "throttling": "PT1S",
+  "notifyConditions": [
+    {
+        "type": "ONCHANGE",
+        "condValues": [
+            "A1", "A2"
+        ]
+    }
+  ]
+}'
+orionCurl --url /v1/subscribeContext?notifyFormat=XML --payload "$payload" --json
+SUB_ID=$(echo "$_response" | grep subscriptionId | awk -F\" '{ print $4}')
+echo
+echo
+
+
+echo "05. Create E1/T1/A1+A2"
+echo "======================"
+payload='{
+  "contextElements": [
+    {
+      "type": "T1",
+      "id":   "E1",
+      "attributes": [
+        {
+          "name": "A1",
+          "type": "string",
+          "value": "1"
+        },
+        {
+          "name": "A2",
+          "type": "string",
+          "value": "2"
+        }
+      ]
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+
+echo "06. Dump accumulator, see E1/T1/A1+A2, in XML"
+echo "============================================="
+curl localhost:${LISTENER_PORT}/dump -s -S 2> /dev/null
+echo
+echo
+
+
+echo "07. PUT /v1/contextSubscriptions/SUB_ID?notifyFormat=JSON, removing A2 from notifyConditions"
```

Comment:
I didn't realize that part 04 works in cross-format way (text: "04. Subscribe to E1/T1/A1+A2, in XML" doesn't mention it and I usually only pay attention to the test outline). Thus, I think is ok as it is.

NTC